### ISSUE TITLE
Upgrade fails with blank page

### ISSUE THREAD

**Issue by leoandrews** [modxbot/migrate#4498]
leoandrews created Redmine issue ID 4498

Upgrading from 2.0.4-pl directly to 2.0.8-pl appears to fail almost at the first hurdle. After asking whether you wish to upgrade or do a full install (and you select upgrade) but this page /setup/index.php?action=options returns an error 500 and on a second attempt, this page: /setup/index.php?action=summary just comes back completely blank.

Everything that should be writeable, according to the loose documentation, is writeable so it's not clear why this is happening

**Comment by opengeek** [comment]
opengeek submitted:

You'll have to provide a lot more information—this is not reproducible in my attempts. You might want to use a forum thread to explore the steps you are using for upgrade first and see if you can get help narrowing down the issue you are experiencing. Once we get more information, we can try and reproduce again...

**Comment by leoandrews** [comment]
leoandrews submitted:

Sure,

The platform is a shared hosting environment running LAMP with PHP at version 5.2.14, MySQL at 5.0.67 with the PDO stuff installed.

Install path was to closely follow the readme (http://rtfm.modx.com/display/revolution20/Upgrading+MODx)  Files were uploaded, overwriting the existing versions with the exception of core/config/config.inc.php as stated in the above help file.

Upon visiting /setup/ the first screen to select the language appears - that's all fine. The second screen which asks you the type of upgrade you require displays without problem either. But after selecting your option, the page first raised a 500 error and second time round came back blank.

So I checked the modx log - nothing. Apache Error log - nothing. The only unusual config setting is in the .htaccess where we've enabled the lines
php_flag zlib.output_compression On
php_value zlib.output_compression_level 5

But running under 2.0.4, this has not had problems.

We're at a dead loss with this one - these white screens tell us nothing, but what's worse is that no log entries are being written. I'd be open to allowing yourself or someone equally expert, FTP access to the site to attempt to reproduce this problem.

One thing that did cross my mind was whether upgrading from 0.4 to 0.8 was too big a step? Perhaps a series of incremental upgrades 0.4-0.5, 0.5-0.6 etc, is a better approach?
